app/src/routes/v2/template.js

```diff
@@ -0,0 +1,76 @@
+const Problem = require('api-problem');
+const templateRouter = require('express').Router();
+
+const { findAndRender, getFromCache } = require('../../components/carboneCopyApi');
+const FileCache = require('../../components/fileCache');
+const { upload } = require('../../components/upload');
+const { truthy } = require('../../components/utils');
+const { middleware } = require('../../components/validation');
+
+const fileCache = new FileCache();
+
+/** Returns the rendered report from cache */
+templateRouter.post('/', upload, async (req, res) => {
+  console.log('Template upload');
```

Comment:
Should we remove this console.log?

Reply:
This is an artifact from npm package shuffling. If/when we get to json log refactor, I'm expecting this to be emitted as a log.verbose notification.